tests/core/config/checkers/test_migration_config_checker.py

```diff
@@ -24,14 +24,6 @@ def mock_func():
 
 
 def test_check_if_entity_property_key_used_is_predefined(caplog):
-    with patch("sys.argv", ["prog", "--production", "1.0"]):
-        core = Core()
-        core.run()
-    assert caplog.text == ""
-    core.stop()
-
-    caplog.clear()
-
```

Comment:
If I understand correctly, this tests what happens when we run the Core service without any config.
Why do we need to remove this test?